assets/src/blocks/PostsListTest/template.js

```diff
@@ -0,0 +1,95 @@
+export const POSTS_LIST_BLOCK_NAME = 'planet4-blocks/posts-list';
+export const POSTS_LISTS_LAYOUT_TYPES = [
+  {label: 'List', value: 'default', columnCount: 3},
+  {label: 'Grid', value: 'grid', columnCount: 4},
+  {label: 'Carousel', value: 'flex', columnCount: 8},
+];
+
+const {__} = wp.i18n;
```

Comment:
What do you think if we keep dependencies to the top of the file and then "custom" `consts`?

Reply:
Hey Dan 👋 You probably missed it, but we kinda abandoned this implementation idea because it doesn't work for the Related Articles 😢 It works in the code editor though, so it seems that the issue is in the frontend rendering